client/landing/stepper/declarative-flow/internals/steps-repository/sensei-purpose/index.tsx

```diff
@@ -0,0 +1,162 @@
+import { TextControl } from '@wordpress/components';
+import { useState } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import React, { useEffect } from 'react';
+import DocumentHead from 'calypso/components/data/document-head';
+import FormattedHeader from 'calypso/components/formatted-header';
+import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
+import { SenseiStepContainer } from '../components/sensei-step-container';
+import { SenseiStepProgress } from '../components/sensei-step-progress';
+import PurposeItem from './purpose-item';
+import type { Step } from '../../types';
+import './style.scss';
+
+const wait = ( ms: number ) => new Promise( ( res ) => setTimeout( res, ms ) );
+
+export const purposes = [
+	{
+		id: 'sell_courses',
+		label: __( 'Sell courses and generate income' ),
+		plugin: { slug: 'woocommerce', id: 'woocommerce/woocommerce' },
+		//description: __( 'WooCommerce will be installed for free.' ),
```

Comment:
Commented out code

Reply:
Also cleaned this in 478d920. I only removed these description temporarily, since the next PR will be doing the installing :)